hazelcast/src/main/java/com/hazelcast/instance/impl/Node.java

```diff
@@ -541,8 +551,11 @@ public String toString() {
             });
             futures.add(future);
         }
+
         try {
-            waitWithDeadline(futures, maxWaitSeconds, TimeUnit.SECONDS, FutureUtil.RETHROW_EVERYTHING);
+            if (!interruptGracefulShutdownRequestFlag.get()) {
+                waitWithDeadline(futures, maxWaitSeconds, TimeUnit.SECONDS, FutureUtil.RETHROW_EVERYTHING);
```

Comment:
My personal preference is to use static imports to reduce the width of the code; the names of values already describe the purpose. Don't feel obligated to use static important, but just think about it.